src/api.ts

```diff
@@ -653,6 +653,37 @@ export class APIGatewayVTL extends VTL {
     return super.eval(node as any, returnVar);
   }
 
+  /**
+   * Attempt to return the expression as a valid escaped json string.
```

Comment:
Does it fail sometimes?

Reply:
No idea. Don't have exhaustive tests or runtime tests, but it works for the two cases I tried. I assume it will need to get more complex as more cases are found or we re-architect to support the various output cases (ex: late resolution with a sub-ast instead of in place transform that can account for the need to be a string instead of an object.)